docs/source/querying_data/remote_processors.md

```diff
@@ -0,0 +1,83 @@
+# Remote Search Processors
+
+> 👉 This page assumes you’re familiar with OpenSearch’s concept of a search pipeline and search processor
```

Comment:
This is not a friendly customer experience. Instead, write a note guiding somewhere were to go to learn.